Scripts/Core/EditorVR.Tools.cs

```diff
@@ -284,12 +299,13 @@ static void DespawnTool(DeviceData deviceData, ITool tool)
 						topTool = deviceData.toolData.Peek();
 						deviceData.currentTool = topTool.tool;
 
-						// Pop this tool of any other stack that references it (for single instance tools)
 						foreach (var otherDeviceData in evr.m_DeviceData)
 						{
 							if (otherDeviceData != deviceData)
 							{
-								if (otherDeviceData.currentTool == tool)
+								// Pop this tool of any other stack that references it (for single instance, multi-device tools)
```

Comment:
Off any other stack

Reply:
Is the multi-device comment here still valid with the new section below?

Reply:
Yes. That's why I distinguished between single-instance, multi-device, and IMultiDeviceTool.

The former is a special case which doesn't really exist anymore, but might be the case for an exclusive mode tool, or may eventually be a use case we want to support. Transform and Locomotion used to be single instance tools.

IMultiDeviceTool is basically just a signal to SpawnTool that it should go on both hands at once.